internal/app/traefik-github-oauth-server/config.go

```diff
@@ -19,16 +19,28 @@ type Config struct {
 	GithubOauthScopes       []string
 }
 
+func envString(key string) string {
+	fileKey := key + "_FILE"
+	if value := os.Getenv(fileKey); value != "" {
+		content, err := os.ReadFile(value)
+		if err != nil {
+			return os.Getenv(key)
+		}
+		return strings.TrimSpace(string(content))
+	}
+	return os.Getenv(key)
+}
+
```

Comment:
looks good to me, would you mind a minor suggestion on the code?

```suggestion
func envFromFile(key string) string {
  fileEnvKey := key + "_FILE"
  
  if value := os.Getenv(fileKey); value != "" {
    content, err := os.ReadFile(value)
    if err == nil  {
      return strings.TrimSpace(string(content))
    }
  }

  return ""
}

// Read from file first, then fallback to read from ENV
func envString(key string) string {
  if value := envFromFile(key); value != "" {
    return value
  }

  return os.Getenv(key)
}

```

Just splitting what the function does

Reply:
Sounds good!